app/views/master_files/_mejs4_player_js.html.erb

```diff
@@ -12,4 +12,5 @@ var mejs4AvalonPlayer = new MEJSPlayer({
   features: ['playpause', 'current', 'progress', 'duration', 'volume', 'quality', 'fullscreen'],
   highlightRail: true
 });
+	currentPlayer = mejs4AvalonPlayer.player;
```

Comment:
I don't think there should be an indentation here?

Reply:
removed indentations